mechanics/TOC-REVIEW.schedule

```diff
@@ -2,10 +2,7 @@
 #@ duration: 45m
 
 # To generate reminders (on a mac):
-# awk -F \| '/^[0-9]/ {cmd "/bin/date -j -f %FT%T " substr($1, 0, 19) " +%e\\ %b" | getline date; \
-#    cmd "/bin/date -j -f %FT%T -v-4d " substr($1, 0, 19) " +%b\\ %e" | getline reminder; \
-#    print "/remind #wg-leads " $2 " is presenting at TOC on " date " (this week) :heart: " reminder}' \
-#    mechanics/TOC-REVIEW.schedule
+# awk -F \| '/^[0-9]/ {cmd "/bin/date -j -f %FT%T " substr($1, 0, 19) " +%e\\ %b" | getline date; cmd "/bin/date -j -f %FT%T -v-4d " substr($1, 0, 19) " +%b\\ %e" | getline reminder; print "/remind #wg-leads " $2 " is presenting at TOC on " date " (this week) :heart: " reminder}' mechanics/TOC-REVIEW.schedule
```

Comment:
wondering what this change does/is for?

Reply:
It's for making copy& paste easier, otherwise you have to step away the comment leftovers manually